src/networking.c

```diff
@@ -3343,9 +3348,9 @@ void readToQueryBuf(client *c) {
 
     c->nread = connRead(c->conn, c->querybuf + qblen, readlen);
     if (c->nread <= 0) {
-        return;
+        return ret;
     }
-
+    ret = (size_t)c->nread == readlen;
```

Comment:
(Nit) The error cases don't really need to return the variable. They can just return `false`. We can declare the variable where we actually need it. (Maybe it's just personal preference. Ignore it if you don't agree.)

```c
    if (c->nread <= 0) {
        return false;
    }

    bool full_read = (size_t)c->nread == readlen;
```